components/script/dom/eventtarget.rs

```diff
@@ -453,7 +454,7 @@ impl EventTarget {
                             args.as_ptr(),
                             body.as_ptr(),
                             body.len() as size_t,
-                            handler.handle_mut())
+                            handler.handle_mut().into())
```

Comment:
What about calling the wrapper function?

Reply:
`handler` is a MutableHandleFunction, a type which has never been a part of `js::rust`.

Reply:
This can be improved along with the HandleString in a followup PR.